public/stylesheets/cards.css

```diff
@@ -72,3 +73,22 @@
 #result-div {
   display: none;
 }
+.hidden-game-data {
+  display: none;
+}
+.guess-buttons {
+  width: 100%;
+  display: flex;
+}
+#page-counter {
+  white-space: nowrap;
+  font-size: 25px;
+  width: auto;
```

Comment:
`auto` should be a default value for `width` attribute I think

Reply:
Will remove it if it is unnecessary. Also, just a heads up, will most likely end up changing styling of these later on. Only tried to add some basic styling to make the results appear easy to read for now.